lib/prompts/createProjectPrompt.ts

```diff
@@ -117,10 +138,19 @@ export async function createProjectPrompt(
 
   if (providedTemplateIsValid) {
     result.projectTemplate = findTemplateByNameOrLabel(
-      projectTemplates,
+      projectTemplates!,
       promptOptions.template!
     );
   }
 
+  if (projectTemplates && projectTemplates.length > 0) {
+    if (!result.projectTemplate) {
+      throw new Error(
+        'Project template is required when projectTemplates is provided'
```

Comment:
Should we move this to the `en.ts`?

Reply:
Good find, moved in [df00fca](https://github.com/HubSpot/hubspot-cli/pull/1449/commits/df00fca88f867c909716a04776e8b2df7327025b)